code/modules/mob/mob_movement.dm

```diff
@@ -345,7 +345,7 @@
 			return FALSE
 		m_intent = new_intent
 	else
-		if(m_intent == MOVE_INTENT_RUN)
+		if(m_intent == MOVE_INTENT_WALK)
```

Comment:
that doenst need changed

Reply:
Isn't that how you change the default move intent for the species? Seems kind of necessary given their intent.